### PR TITLE
Hotfix: Removed "bug" in Vault policy & upated readme

### PR DESCRIPTION
**Pull request by claesgill**
## Closes/fixes/resolves issue(s)?
No issue - hotfix 🔥 

## What was added/changed/fixed?
- Fixed `failed_when` bug in ansible-playbook
- Minor restructure of README.

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,16 +48,23 @@ Terraform-nomad-minio module is IaC - infrastructure as code. Module contains a
...
-### Providers
-- [Nomad](https://registry.terraform.io/providers/hashicorp/nomad/latest/docs)
+### Verifying setup
+You can verify successful run with next steps:
+
+* create local proxy to minio instance with `consul` binary. Check [required software section](#required-software)
+
+```text
+make proxy
+```
```
Comment (pdmthorsrud) [inline README.md]: Add section on how to access the UI

**Review by pdmthorsrud — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -40,24 +39,33 @@ Terraform-nomad-minio module is IaC - infrastructure as code. Module contains a
...
 
 #### Required software
 - [GNU make](https://man7.org/linux/man-pages/man1/make.1.html)
-- [consul](https://releases.hashicorp.com/consul/) binary available on `PATH` on the local machine.
+- [Consul](https://releases.hashicorp.com/consul/) binary available on `PATH` on the local machine.
 
 #### Other
 
 ## Usage
+The following command will run an example with standalone instance of minio.
```
Comment (dangernil) [inline README.md]:
```suggestion
The following command will run an example with standalone instance of Minio.
```

Maybe be consistent with uppercase. Should check the whole document for any missed namings.

Comment (pdmthorsrud) [reply README.md]: Actually, it was a discussion in mattermost the other day, where it was mentioned we should use "egennavn"! So it should be MinIO

EDIT: Oh my god, I've become a "Well, actually ..." dude.

**Review by dangernil — DISMISSED** [review]
LGTM

**Review by pdmthorsrud — APPROVED** [review]

**Review by zhenik — APPROVED** [review]

**Pull request merged**